__tests__/database.test.js

```diff
@@ -235,20 +235,20 @@ describe("POST comments", () => {
 })
 
 describe("PATCH articles", () => {
+    const matchArticleObject = {
```

Comment:
just to make sure each test is isolated and isn't affected by any other test, we should try to avoid the sharing of variables - always declare them new within each test